### PR TITLE
[Search] Create search UI

### PR DESCRIPTION
**Pull request by slhale**
Was for deployment on the demo site.

### CODE REVIEW

File: bundles.json [hunk 1]
```diff
@@ -18,13 +18,13 @@
...
     "platform/forms",
+    "platform/persistence/elastic",
     "platform/persistence/queue",
     "platform/policy",
     "platform/entanglement",
     "platform/search",
 
     "example/imagery",
-    "example/persistence",
     "example/eventGenerator",
```
Comment (VWoeltjen) [inline bundles.json]: @slhale Can we revert this change before merge (e.g. stick with example/persistence in the default configuration?)

Alternately, I'd be okay with `example/localstorage` at this point (and think that is reasonably relevant here since it provides an initial set of domain objects to index for search...)

Comment (slhale) [inline bundles.json]: @VWoeltjen I will change it back before merge. It's just useful for testing while in development to have ES.

Comment (larkin) [inline bundles.json]: @slhale note that in development you can run the server with `node app.js -x example/persistence -i platform/persistence/elastic` to make these same changes without editing files.

Comment (slhale) [inline bundles.json]: @larkin Is there a way to do that with http-server?

Comment (larkin) [inline bundles.json]: aah, nope, but it ****should**** be really easy to switch from http-server to the above; just `npm install` and then run the above command.

Comment (slhale) [inline bundles.json]: That worked. Thanks.

**Comment by slhale** [comment]
Author Checklist
1. Changes address original issue? Y
2. Unit tests included and/or updated with changes? Y
3. Command line build passes? Y
4. Expect to pass code review? Y
5. Project-specific information isolated to appropriate branches? Y

**Comment by VWoeltjen** [comment]
Looks good!

1. Changes appear to address issue? Y
2. Appropriate unit tests included? Y
3. Code style and in-line documentation are appropriate? Y
4. Commit messages meet standards? Y
